src/App.js

```diff
@@ -82,35 +73,20 @@ class App extends Component {
       transactions: [],
       blocks: [],
     };
-
-    const self = this;
-
-    // update global info (once, on load)
-    this.getRemoteState(
-      'globalStats',
-      `http:${BLOCK_EXPLORER_API_BASE}/global-stats`,
-    );
-
-    // update cluster info (once, on load)
-    this.getRemoteState(
-      'clusterInfo',
-      `http:${BLOCK_EXPLORER_API_BASE}/cluster-info`,
-      null,
-      null,
-      self.parseClusterInfo,
+    this.state = Object.assign(
+      {
+        enabled: true,
+        dialogOpen: false,
+        ws: null,
+      },
+      this.defaultState,
     );
 
-    // update blocks (once, on load)
-    self.updateBlocks();
-
-    self.updateTxnStats();
     setInterval(() => {
```

Comment:
these setInterval() calls will likely also become part of the state when we implement configurable delay